src/components/App/App.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { CssVariables, CssReset } from '@dhis2/ui'
+import { FAVORITE_VIEWS } from '../../constants/categories.js'
+import { WEEK } from '../../constants/intervals.js'
+import { SUM } from '../../constants/aggregations.js'
+import { ALL } from '../../constants/chartTypes.js'
+import { CHART_VIEW } from '../../constants/eventTypes.js'
+import { PS_25 } from '../../constants/pageSizes.js'
+import { ASC } from '../../constants/sortOrders.js'
+import Filters from '../Filters/index.js'
+import {
+    LayoutContainer,
+    LayoutSidebar,
+    LayoutContent,
+} from '../Layout/index.js'
+import CategoryField from '../CategoryField/index.js'
+import AppTitle from '../AppTitle/index.js'
+import Visualization from '../Visualization/index.js'
+import './App.css'
+import createDefaultDates from './createDefaultDates.js'
+
+const App = () => {
+    const { initialStartDate, initialEndDate } = createDefaultDates()
+
+    // State
+    const [aggregation, setAggregation] = useState(SUM)
+    const [category, setCategory] = useState(FAVORITE_VIEWS)
+    const [chartType, setChartType] = useState(ALL)
+    const [endDate, setEndDate] = useState(initialEndDate)
+    const [eventType, setEventType] = useState(CHART_VIEW)
+    const [interval, setInterval] = useState(WEEK)
+    const [pageSize, setPageSize] = useState(PS_25)
+    const [sortOrder, setSortOrder] = useState(ASC)
+    const [startDate, setStartDate] = useState(initialStartDate)
```

Comment:
*minor* this is a LOT of state variables... should we change this to either a single `useState` object or even a `useReducer`?

Reply:
I came across this recommendation: https://reactjs.org/docs/hooks-faq.html#should-i-use-one-or-many-state-variables. The recommendation is:

> we recommend to split state into multiple state variables based on which values tend to change together

Since they are all separate controls none of them really change together. So I do see your point of this being quite a lot of state variables, but on the other hand, it does seem to be the simplest approach to me. I don't really mind the amount _that_ much, but that could also just be me.

I postponed this for now: https://github.com/dhis2/notes/pull/181, so that we can tackle it later in a separate PR.